github/installation.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
I agree this seems to fit really nicely right under the definition of `IntegrationEvent` that first mentions it. It doesn't seem like a win to break it out into a separate tiny file.

> I was merely trying to follow the structure of the other events

I don't quite see where that's the case, can you point it out please?